music_assistant/server/providers/soundcloud/__init__.py

```diff
@@ -0,0 +1,338 @@
+"""Soundcloud support for MusicAssistant."""
+import asyncio
+
+# from urllib.parse import unquote
+from typing import AsyncGenerator, Callable  # noqa: UP035
```

Comment:
why the noqa ?

Reply:
I even dont know what this is, it's a leftover from another file

Reply:
I removed it